modules/lvgl/include/lvgl_common_input.h

```diff
@@ -36,8 +36,9 @@ int lvgl_init_input_devices(void);
 #define LVGL_KEY_VALID(key)     IN_RANGE(key, 0, UINT8_MAX)
 
 #define LVGL_INPUT_DEFINE(inst, type, msgq_size, process_evt_cb)                                   \
-	INPUT_CALLBACK_DEFINE(LVGL_INPUT_DEVICE(inst), process_evt_cb,                             \
-			      (void *)DEVICE_DT_INST_GET(inst));                                   \
+	INPUT_CALLBACK_DEFINE_NAMED(LVGL_INPUT_DEVICE(inst), process_evt_cb,                       \
+				    (void *)DEVICE_DT_INST_GET(inst),                              \
+				    _input_callback__process_evt_cb__##inst);                      \
```

Comment:
```suggestion
				    process_evt_cb__##inst);                      \
```

the macro already adds a `_input_callback__`, no need to double up on it

Reply:
Does it? I thought only `INPUT_CALLBACK_DEFINE()` does that. 🤔 

Does `INPUT_CALLBACK_DEFINE()` then duplicate the `_input_callback` prefix?

Reply:
oh, hey you are right it doesn't! it should though, would you mind adding a commit to fix that while you are at this? remove the prefix from https://github.com/zephyrproject-rtos/zephyr/blob/main/include/zephyr/input/input.h#L159 and move it up to https://github.com/zephyrproject-rtos/zephyr/blob/main/include/zephyr/input/input.h#L140 so that the APIs are coherent

Reply:
Done. I added the change to this PR, if I should instead open a new PR, let me know.

Reply:
No that's great, thanks a lot!